autorosters/autorosters_main.py

```diff
@@ -204,6 +204,14 @@ def process_game_data(self):
                             player["games_by_role"][role] += f"{'n' * math.ceil(int(match['best_of']) / 2)},"
                 continue
             for game in match["games"].values():
+                if game["msg_data"]["MSGFF"]:
```

Comment:
I'll merge this anyway cos it doesn't really matter, but generally it's good to explicitly write `is not None` (since now we're on a Cargo version where it'll be None otherwise)